db/migrate/20241023082117_create_active_storage_tables.active_storage.rb

```diff
@@ -0,0 +1,57 @@
+# This migration comes from active_storage (originally 20170806125915)
+class CreateActiveStorageTables < ActiveRecord::Migration[7.0]
+  def change
+    # Use Active Record's configured type for primary and foreign keys
+    primary_key_type, foreign_key_type = primary_and_foreign_key_types
+
+    create_table :active_storage_blobs, id: primary_key_type do |t|
+      t.string   :key,          null: false
+      t.string   :filename,     null: false
+      t.string   :content_type
+      t.text     :metadata
+      t.string   :service_name, null: false
+      t.bigint   :byte_size,    null: false
+      t.string   :checksum
+
+      if connection.supports_datetime_with_precision?
+        t.datetime :created_at, precision: 6, null: false
```

Comment:
purpose of the using precisions as 6?

Reply:
Bhai, using precision as 6 for datetime enables microsecond-level timestamps. Also, PostgreSQL supports storing datetime with fractional seconds, which can be beneficial for compatibility and consistency.